### ISSUE TITLE
Minecraft keeps crashing

### ISSUE THREAD

**Issue by ZacharyWithMinecraft** [LambdAurora/LambDynamicLights#204]
so when i open minecraft its fine on the title screen but when i open an world it first autosaves and then crashes

**To Reproduce**
i actually have no idea it just crashes

**Expected behavior**
i expected my world to open normally

**Desktop (please complete the following information):**
 - OS: [Windows 10 home]
 - Minecraft [1.20.4]
 - Fabric Loader version [ 0.15.3+build:idk]
 - Mods [i have 113 so i cant write them here]
 - Version [WHAT VERSION!?!]
 - Branch [whats a branch]

**Comment by ZacharyWithMinecraft** [comment]
plz help guys
i beg

**Comment by tocram1** [comment]
I'm also having this crash, with this exception:
```
Description: Exception ticking world

org.spongepowered.asm.mixin.injection.throwables.InjectionError: LVT in net/minecraft/class_1937::method_18471()V has incompatible changes at opcode 81 in callback lambdynlights.mixins.json:WorldMixin from mod lambdynlights->@Inject::onBlockEntityTick(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Lnet/minecraft/class_3695;Ljava/util/Iterator;Lnet/minecraft/class_5562;)V.
 Expected: [Lnet/minecraft/class_3695;, Ljava/util/Iterator;, Lnet/minecraft/class_5562;]
    Found: [Lnet/minecraft/class_3695;, Ljava/util/Iterator;, Z]
Available: [Lnet/minecraft/class_3695;, Ljava/util/Iterator;, Z, Lnet/minecraft/class_5562;]
	at net.minecraft.class_1937.handler$bdj000$lambdynlights$onBlockEntityTick$missing(class_1937.java)
	at net.minecraft.class_1937.method_18471(class_1937.java:482)
	at net.minecraft.class_3218.method_18765(class_3218.java:404)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:948)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:845)
	at net.minecraft.class_1132.method_3748(class_1132.java:110)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:683)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

I get these lines in startup logs :
```
22:02:17.144    FabricLoader/Mixin    main
Injection error: LVT in net/minecraft/class_1937::method_18471()V has incompatible changes at opcode 81 in callback lambdynlights.mixins.json:WorldMixin from mod lambdynlights->@Inject::onBlockEntityTick(Lorg/spongepowered/asm/mixin/injection/callback/CallbackInfo;Lnet/minecraft/class_3695;Ljava/util/Iterator;Lnet/minecraft/class_5562;)V.
 Expected: [Lnet/minecraft/class_3695;, Ljava/util/Iterator;, Lnet/minecraft/class_5562;]
    Found: [Lnet/minecraft/class_3695;, Ljava/util/Iterator;, Z]
Available: [Lnet/minecraft/class_3695;, Ljava/util/Iterator;, Z, Lnet/minecraft/class_5562;]

22:02:35.729    lambdynlights    Render thread
[LambDynLights] Initializing LambDynamicLights...

22:02:35.769    lambdynlights    Render thread
[LambDynLights] Configuration loaded.
```

Here's my mods list:
- Dynamic FPS
- Indium
- Iris
- LambDynamicLights
- Lithium
- Mod Menu
- Nvidium (unused)
- Sodium
- Sodium Extra
- Xaero's Minimap

I'm also on Minecraft 1.20.4, with Fabric 0.15.3.
I'm trying to pinpoint which mods breaks compatibility with LambDynamicLights, and I'll come back here if I find anything useful.

**Comment by tocram1** [comment]
Okay this will much definitely be fixed with #202

**Comment by DovydasTEDS** [comment]
This will be fixed when #202 is merged into the main branch.

**Comment by LambdAurora** [comment]
Fixed in 2.3.4+1.20.4
